src/state/reducers/rootReducer.js

```diff
@@ -0,0 +1,28 @@
+const rootReducer = (state, action) => {
+	switch (action.type) {
+		case 'SUCCESS_MESSAGE':
+			return {
+				...state,
+				message: action.payload,
+				successOpen: true,
+			};
+
+    case 'CLOSE_MESSAGE': 
+    return {
+      ...state,
+      successOpen: false,
+      errorOpen: false
+    }
+    case 'ERROR_MESSAGE':
+      return {
+        ...state,
+        message: action.payload,
+				errorOpen: true
+      }
+
+		default:
+			return state;
+	}
+};
+
+export default rootReducer;
```

Comment:
Please fix the formatting on this file here on GH (and please refrain from saying "it looks good in my IDE")

Reply:
done